src/templates/next/components/internal/Card.tsx

```diff
@@ -43,26 +43,35 @@ const TextComponent = ({
 }) => {
   return (
     <div
-      className={`py-6 px-5 sm:p-7 flex flex-col flex-grow gap-6 justify-between ${className}`}
+      className={`flex flex-grow flex-col justify-between gap-6 px-5 py-6 sm:p-5 ${className}`}
     >
       <div className="flex flex-col gap-3">
         {title && (
-          <h4 className={`${Heading[4]} text-content-strong line-clamp-2`}>{title}</h4>
+          <h4 className={`${Heading[4]} line-clamp-2 text-content-strong`}>
+            {title}
+          </h4>
         )}
         {text && (
-          <p className="text-content text-left text-base line-clamp-4	sm:text-lg grow">
+          <p className="line-clamp-4 grow text-left text-base	text-content sm:text-lg">
             {text}
           </p>
         )}
       </div>
       {buttonLabel && (
         <div className={`flex items-center gap-1`}>
-          <a href={url}
-      target={url.startsWith("http") ? "_blank" : undefined}
-      rel={url.startsWith("http") ? "noopener noreferrer nofollow" : undefined} className={`text-interaction-link after:absolute after:inset-0 ${ButtonLink[1]}`}>
+          <a
```

Comment:
@isomerpages/iso-engineers fyi this is something i found here > https://inclusive-components.design/cards/. not sure if it makes sense or is it worth doing!

Reply:
actually that's a great article! i think it'd be worth doing _in the long run_ - not sure now is the right time given time constraints :monkas: